README.md

```diff
@@ -49,6 +49,34 @@ Resources
 - [Code Examples](examples)
 - [API reference](https://odlgroup.github.io/odl/odl.html)
 
+Applications
+============
+ODL has been used in a wide range of research and development with several research articles and conference contributions using ODL, as well as other open source projects using ODL as a back-end. Many of these also release their code publicly can serve as examples of how to use ODL. This is a short list of such projects. If you want your project added, contact the developers.
```

Comment:
research articles -> scientific papers
~~using ODL~~
[...] code publicly **and** can [...]
want your project added -> want to add your project to the list
contact the **maintainers or file a pull request**.